alea/model.py

```diff
@@ -16,39 +15,45 @@
 class StatisticalModel:
     """
     Class that defines a statistical model.
-    The statisical model contains two parts that you must define yourself:
-        - a likelihood function, ll(self, parameter_1, parameter_2... parameter_n):
-            a function of a set of named parameters
-            returns a float expressing the loglikelihood for observed data
-            given these parameters
-        - a data generation method generate_data(self, parameter_1, parameter_2... parameter_n):
-            a function of the same set of named parameters
-            returns a full data set:
-
-    Methods:
-        __init__
-        required to implement:
-
-        _ll
-        _generate_data
-
-        optional to implement:
-        get_expectation_values
-
-        Implemented here:
-        store_data
-        fit
-        get_parameter_list
-
-    Other members:
-        _data = None
-        _config = {}
-        _confidence_level = 0.9
-        _confidence_interval_kind = "upper, lower, central"
-            (if your threshold is the FC threshold, "central" gives you the unified interval)
-        _confidence_interval_threshold: function that defines the Neyman threshold for limit calculations
-        _fit_guess = {}
-        _fixed_parameters = []
+
+    - The statisical model contains two parts that you must define yourself:
+        - a likelihood function
+            ll(self, parameter_1, parameter_2... parameter_n):
+            A function of a set of named parameters which
+            return a float expressing the loglikelihood for observed data given these parameters.
+        - a data generation function
+            generate_data(self, parameter_1, parameter_2... parameter_n):
+            A function of the same set of named parameters return a full data set.
+    - Methods that you must implement:
+        - _ll
+        - _generate_data
+    - Methods that you may implement:
+        - get_expectation_values
+    - Methods that already exist here:
+        - ll
+        - store_data
+        - fit
+        - get_parameter_list
+
+    Attributes:
+        data: data of the model
+        _data: data of the model
+        _confidence_level: confidence level for confidence intervals
+        _confidence_interval_kind: kind of confidence interval to compute
+        parameters: parameters of the model
+        confidence_interval_threshold: threshold for confidence interval
+        is_data_set (bool): True if data is set
```

Comment:
I think it only makes sense to list here the ones that we want people to use. I.e. remove the ones with an underscore and `is_data_set`.